packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.scss

```diff
@@ -1,7 +1,7 @@
 $meter-colors: (
     initial: var(--system-light-6-hover-background, #e6e9e9),
     weak: var(--status-light-danger, #ec3f3f),
-    moderate: var(--status-light-warning, #ffad3a),
+    moderate: var(--status-light-danger, #ec3f3f),
```

Comment:
why we have moderate since the behaviour is same as week? can we merge these two?

Reply:
The width is different actually. But I will put the colors in the var